qcfractal/interface/client.py

```diff
@@ -524,7 +527,13 @@ def list_collections(
             df = df[df["visibility"]]
         if group is not None:
             df = df[df["group"].str.lower() == group.lower()]
-        df.drop(["visibility", "group"], axis=1, inplace=True)
+        if tag is not None:
+            if isinstance(tag, str):
+                tag = [tag]
+            tag = {t.lower() for t in tag}
+            df = df[df.apply(lambda x: len({t.lower() for t in x['tags']} & tag) > 0, axis=1)]
```

Comment:
@dgasmith someday, all of dataset will be a pandas one-liner.

Reply:
Or in SQL :)